src/Tribe/Views/V2/Views/By_Day_View.php

```diff
@@ -172,21 +240,21 @@ public function get_grid_days( $date = null, $force = false ) {
 			} else {
 				$start = tribe_beginning_of_day( $day->format( Dates::DBDATETIMEFORMAT ) );
 				$end   = tribe_end_of_day( $day->format( Dates::DBDATETIMEFORMAT ) );
-				/*
-				 * We want events overlapping the current day, by more than 1 second.
-				 * This prevents events ending on the cutoff from showing up here.
-				 */
-				$day_query = tribe_events()
-					->set_found_rows(true)
-					->by_args( $repository_args )
-					->where( 'date_overlaps', $start, $end, null, 2 )
-					->per_page( $events_per_day )
-					->order_by( $order_by, $order );
-				$day_event_ids = $day_query->get_ids();
-				$found     = $day_query->found();
-
-				$this->grid_days_cache[ $day_string ]       = (array) $day_event_ids;
-				$this->grid_days_found_cache[ $day_string ] = (int) $found;
+
+				$results_in_day = array_filter( $day_results, static function( $event ) use ( $start, $end ) {
```

Comment:
* :no_entry_sign: Opening parenthesis of a multi-line function call must be the last content on the line
* :no_entry_sign: Only one argument is allowed per line in a multi-line function call

<sub>hash: 3f10ae0fe370abbb36ab243bc6fbabe7</sub>